src/test/java/org/symphonyoss/symphony/messageml/elements/form/CheckboxTest.java

```diff
@@ -26,6 +26,81 @@ public void beforeEach() {
     this.checked = "false";
   }
 
+  @Test
+  public void testPresentationMLCheckbox() throws Exception {
+    String input = String.format("<div data-format=\"PresentationML\" data-version=\"2.0\">" +
+        "<form>" +
+        "<div class=\"checkbox-group\">" +
+        "<input type=\"checkbox\" name=\"%s\" value=\"%s\"/>" +
+        "<label>%s</label>" +
+        "</div></form></div>", this.name, this.value, this.text);
+
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+    verifyMessageMLObjectsForCheckbox(context);
```

Comment:
The way you're verifying the output makes it quit hard to understand what the expected PresentationML, Markdown and text is. I would be nice if the expected outputs were right there in the test methods.

Reply:
I'm not sure with this one as this would require kind-of duplicated code. Also, I don't think it's that hard to understand, I mean, it's just a bunch of String format to parameterize the check of the parameters